### PR TITLE
Adds specific text style mixins

### PR DESCRIPTION
**Pull request by finnclark**
cc @andycarrell 

- Adds mixins for card text. Moved away from h3, h4 etc because it's not really following the structure properly anyways. 
- Swapped some styling over but realised that you're probably changing the structure quite a bit so happy to remove these changes from this PR

Could maybe even remove the h3 and h4 heading mixins if they aren't being used - Just keep h1 and h2?

### CODE REVIEW

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "../../styles/vars.scss";
+@import "../../styles/typography.scss";
```
Comment (finnclark) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:2]: mentioned in the description but I can remove the changes from this file if that's preferred

Comment (andycarrell) [reply app/components/DosesDescriptionList/DosesDescriptionList.module.scss:2]: Na this is all good! I can rebase ;)

**Comment by andycarrell** [comment]
Some design feedback - just an opinion tho, don't have to change 🙏 

- I think the uppercase text should have a heavier font weight (500 in screenshot)
- Should we align the text to "baseline" across the term, percentage & doses remaining?

image

**Comment by finnclark** [comment]
Sorry my bad, this PR was just to add the mixins for you to rebase into your new layout PR, I'm not actually keen to update the production layout design to use these styles 😊

The font-weight change looks ok but I'd be keen to double check with the new layout when you update your PR if that's ok?

**Comment by andycarrell** [comment]
@finnclark - yup that makes sense, before we merge we can remove the mixins, then it won't ship 🙏 

From an implementation perspective, these classnames are global, but they're very specific - for example `vaccine-count`, it's pretty clear where that's expected to be used, but it's not that clear how we would style the next piece of text that ultimately has the same size / height / weight - so I don't think these should be mixins.

I think there's two options here either we:

- Avoid using mixins for typography and move the specific styling to `DosesDescriptionList.module.scss`. It might also make sense to create a font-size / weight scale that DosesDescription can reference

- Make the mixins more generic, probably by bringing back a scale like we had: heading-1 heading-2 etc in typography, then each component determines which classname it should use. If we're doing that, we probably don't need a mixin since we can just create classNames for each?

**Comment by finnclark** [comment]
Updated per call @andycarrell

**Review by andycarrell — APPROVED** [review]

**Pull request merged**